src/components/ProductCard.tsx

```diff
@@ -139,6 +38,53 @@ const REPO_DATA = gql`
   }
 `
 
+const SubjectPill: React.FC<{ subject: string; children: ReactNode }> = ({
+  subject,
+  children,
+}) => {
+  const bgColor = () => {
+    switch (subject) {
+      case "Solidity":
+        return "tagYellow"
+      case "Vyper":
+        return "tagBlue"
+      case "web3":
+        return "tagTurquoise"
+      case "JavaScript":
+        return "tagRed"
+      case "TypeScript":
+        return "tagBlue"
+      case "Go":
+        return "tagTurquoise"
+      case "Python":
+        return "tagMint"
+      case "Rust":
+        return "tagOrange"
+      case "C#":
+        return "tagBlue"
+      case "Java":
+        return "tagPink"
+      default:
+        return "tagGray"
+    }
+  }
+  return (
+    <Box
```

Comment:
These should be Badges in the upcoming refactor, right?

Reply:
Yes! I have made a note in #8601